tests/acceptance/client_span_test.py

```diff
@@ -43,13 +43,16 @@ def test_headers_created_for_sampled_child_span(sampled_trace_id_generator):
 
     expected = {
         'X-B3-Flags': '0',
-        'X-B3-ParentSpanId': '1',
+        'X-B3-ParentSpanId': '0x1234',
         'X-B3-Sampled': '1',
         'X-B3-TraceId': '0x0',
         }
 
-    headers = TestApp(main({}, **settings)).get('/sample_child_span',
-                                                status=200)
+    with mock.patch('pyramid_zipkin.request_helper.generate_span_id') \
```

Comment:
Why do you have to mock this out? It looks like the code in the test below is just ignoring this value anyways.